### PR TITLE
Auto-generate README with tag information (#192)

### PR DESCRIPTION
**Pull request by malcyon**
Auto-generate the Readme so the releaser doesn't have to manually update each tag in the README.

This is a backport of #192 to the `4.1` branch.

(cherry picked from commit 44ec70953c28d90af5f493f6bb27779c01eeff7d)

### CODE REVIEW

File: version.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+graylog:
+  major_version: 4
+  minor_version: 1
+  patch_version: 9
+  release: 1
+forwarder:
+  version: 4.1
+  release: 5
```
Comment (thll) [inline version.yml:8]: I'm not 100% sure, but do we really have release `5` out there? The highest number I can find in an existing tag was `2`.

```suggestion
  release: 2
```

**Review by thll — CHANGES_REQUESTED** [review]

**Comment by malcyon** [comment]
Just dropping a reminder here for myself. They `jenkins.groovy` pipeline is still referencing the deleted `VERSION` and `                  VERSION_FORWARDER_PACKAGE` files. This will need to be updated before this is merged.

File: version.yml (hunk 1 above)
Comment (malcyon) [reply version.yml:8]: I was just copying what was in the `VERSION_FORWARDER_PACKAGE` file. But you're right, we only have rev 2 published.

**Comment by malcyon** [comment]
Now it's throwing errors trying to build the Forwarder image. I will fix this tomorrow.

File: version.yml (hunk 1 above)
Comment (malcyon) [reply version.yml]: Okay, I was wrong. We do have 4.1 rev 5. Because we had a beta and some release candidates.

**Review by thll — APPROVED** [review]

**Pull request merged**